src/main/webapp/WEB-INF/index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <meta charset="UTF-8">
-  <title>Tasks Scheduler</title>
-  <link href="style.css" rel="stylesheet">
+    <meta charset="UTF-8">
+    <title>Tasks Scheduler</title>
+    <link href="../style.css" rel="stylesheet">
```

Comment:
These dots should not be needed. Instead, all paths should be absolute, that is `href="/style.css"`. This way - regardless of the url address user visits - it would always refer to DOMAIN/style.css

Reply:
Ah, yes, right.